src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.html

```diff
@@ -50,8 +50,16 @@ <h6>
         </div>
         <div *ngIf="result && result.feedbacks && result.feedbacks.length > 0">
             <br />
-            <h5><span jhiTranslate="artemisApp.fileUploadExercise.assessedSubmission">Your assessed submission</span>:</h5>
-            <jhi-file-upload-result [result]="result"></jhi-file-upload-result>
+            <h5><span jhiTranslate="artemisApp.fileUploadExercise.assessedSubmission">Your Assessment</span>:</h5>
+            <div class="row">
+                <jhi-additional-feedback
+                    *ngIf="unreferencedFeedback && unreferencedFeedback.length != 0"
```

Comment:
I think you can simply that and I would use > 0 instead != 0 :) 

```suggestion
                    *ngIf="unreferencedFeedback?.length > 0"
```

Reply:
This is also for the other exercise types

Reply:
Sounds good :)